django/db/models/sql/query.py

```diff
@@ -800,8 +800,8 @@ def change_aliases(self, change_map):
         # "group by" and "where".
         self.where.relabel_aliases(change_map)
         if isinstance(self.group_by, tuple):
-            self.group_by = tuple([col.relabeled_clone(change_map) for col in self.group_by])
-        self.select = tuple([col.relabeled_clone(change_map) for col in self.select])
+            self.group_by = tuple(col.relabeled_clone(change_map) for col in self.group_by)
```

Comment:
Ah yes, I remember that. It seems to be generally the case, even with a range of 100k items:

```
In [1]: %timeit tuple(x for x in range(100000))
100 loops, best of 3: 9.75 ms per loop

In [2]: %timeit tuple([x for x in range(100000)])
100 loops, best of 3: 7.04 ms per loop
```

The difference *is* tiny, the main point of #7730 and my related PR's is to optimize the often repeated code paths like `_clone`, not these individual operations. I changed it there just because of the comment.

Anyway we could just turn off the rule in `flake8-comprehensions`